scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -113,6 +113,7 @@ function doCheckSuccessOfPluginInstall(){
 # return: dependencies that are already installed
 function doCheckPluginsInstalled() {
```

Comment:
Does this check really work? I can continuously add the same plugin via `add-plugin` and get the same output that the plugin is successfully installed. How does the uninstallation process of eclipse work? It seems that the plugin is not removed in `plugins` directory when using 

`debugInfo=$(doInvokeEclipse org.eclipse.equinox.p2.director -uninstallIU "${plugin_id}")`

instead it will be removed in `configuration/org.eclipse.equinox.simpleconfigurator/bundles.info` file.